integration_test/uninstall_test.go

```diff
@@ -47,6 +47,23 @@ func TestKrewUninstall(t *testing.T) {
 	}
 }
 
+func TestKrewUninstallPluginsFromCustomIndex(t *testing.T) {
```

Comment:
thanks for adding these.

Reply:
`TestKrewUninstal_CannotUseIndexSyntax` is a better name though. feel free to mix.